### PR TITLE
Add commutation n timing

### PR DESCRIPTION
**Pull request by damosvil**

### CODE REVIEW

File: _pages/CalcNextCommPeriod.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: CalcNextCommPeriod routine
```
Comment (stylesuxx) [inline _pages/CalcNextCommPeriod.md]: Titles should be as routines are called in code, so people can easily find them ```calc_next_comm_period```. I also added a filter to directly link to searchresults for the code, but i can add that myself ;-)

File: _pages/InitializeTiming.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: InitializeTiming routine
```
Comment (stylesuxx) [inline _pages/InitializeTiming.md]: ```initialize_timing```

File: _pages/WaitForTimer3.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: WaitForTimer3 MACRO
```
Comment (stylesuxx) [inline _pages/WaitForTimer3.md]: ```Wait_For_Timer3```

**Review by stylesuxx — CHANGES_REQUESTED** [review]

File: _pages/CalcNextCommPeriod.md (hunk 1 above)
Comment (damosvil) [reply _pages/CalcNextCommPeriod.md]: Fixed

File: _pages/InitializeTiming.md (hunk 2 above)
Comment (damosvil) [reply _pages/InitializeTiming.md]: Fixed

File: _pages/WaitForTimer3.md (hunk 3 above)
Comment (damosvil) [reply _pages/WaitForTimer3.md]: Fixed

**Review by stylesuxx — APPROVED** [review]

**Pull request merged**